### DISCUSSION TITLE
isEqual documentation is not available

### DISCUSSION THREAD

**Discussion by touhidrahman** [rayepps/radash#162] [General]
I just noticed that the `isEqual` was implemented a while back ([PR](https://github.com/rayepps/radash/pull/53)), but the documentation was not updated. I looked for it, and it seems the doc is not publicly editable. So creating this discussion as a reminder. Thank you.

**Comment by sodiray** [comment]
Hey @touhidrahman thanks for bringing this up. The truth is this library surged in popularity (and community contributions) and I wasn't ready for that at all. Turns out 80% of the work is not development. I'm currently working on the docs to make them versioned and up to date 👍 fast as I can 😅 

The docs are public tho, you can see them in the [radash-docs repo](https://github.com/rayepps/radash-docs). If you wanted to add a new doc page you would create a new file in [this directory](https://github.com/rayepps/radash-docs/tree/master/src/pages/docs).
